RELEASE/scripts/autoscend/paths/pocket_familiars.ash

```diff
@@ -1,11 +1,16 @@
+boolean in_pokefam()
```

Comment:
This looks suspicious

Reply:
oh, ha. I accidentally changed this when doing find and replace. that is funny